nextcloudappstore/core/models.py

```diff
@@ -88,34 +89,52 @@ def can_update(self, user: User) -> bool:
     def can_delete(self, user: User) -> bool:
         return self.owner == user
 
-    def releases_by_platform_v(self):
+    def releases_by_platform_v(self, nightlies=False):
```

Comment:
It's not inclusion, but exclusive selection, see the docstring. But yes, I'm open to the idea that there might be a better name.